easybuild/easyblocks/o/openfoam.py

```diff
@@ -181,6 +181,8 @@ def configure_step(self):
 
         # inject compiler variables into wmake/rules files
         ldirs = glob.glob(os.path.join(self.builddir, self.openfoamdir, 'wmake', 'rules', 'linux*'))
+        if LooseVersion(self.version) >= LooseVersion('v1906'):
```

Comment:
This comparison is only meaningful if the version is also of the form `v[0-9]`, so make sure that's the case?

```python
if self.version.startswith('v') and LooseVersion(self.version) >= LooseVersion('v1906'):
```

Reply:
done